omnibus/config/software/datadog-trace-agent.rb

```diff
@@ -32,27 +32,29 @@
   gopath = Pathname.new(project_dir) + '../../../..'
   if windows?
     env = {
-        'GOPATH' => gopath.to_path,
-        'PATH' => "#{gopath.to_path}/bin:#{ENV['PATH']}",
-        'WINDRES' => 'true',
+      'GOPATH' => "#{windows_safe_path(gopath.to_path)}",
```

Comment:
Please bring along the comment so we know why the windows steps are different from Linux, for next time we have to chase down build breakages.